src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -867,366 +855,202 @@ void SetUpZoneSizingArrays(EnergyPlusData &state)
-                calcZoneSizing.DesCoolMinAirFlow = zoneSizingInput.DesCoolMinAirFlow;
-                calcZoneSizing.DesCoolMinAirFlowFrac = zoneSizingInput.DesCoolMinAirFlowFrac;
-                calcZoneSizing.InpDesHeatAirFlow = zoneSizingInput.DesHeatAirFlow;
-                calcZoneSizing.DesHeatMaxAirFlowPerArea = zoneSizingInput.DesHeatMaxAirFlowPerArea;
-                calcZoneSizing.DesHeatMaxAirFlow = zoneSizingInput.DesHeatMaxAirFlow;
-                calcZoneSizing.DesHeatMaxAirFlowFrac = zoneSizingInput.DesHeatMaxAirFlowFrac;
-                calcZoneSizing.HeatSizingFactor = zoneSizingInput.HeatSizingFactor;
-                calcZoneSizing.CoolSizingFactor = zoneSizingInput.CoolSizingFactor;
-                calcZoneSizing.AccountForDOAS = zoneSizingInput.AccountForDOAS;
-                calcZoneSizing.DOASControlStrategy = zoneSizingInput.DOASControlStrategy;
-                calcZoneSizing.DOASLowSetpoint = zoneSizingInput.DOASLowSetpoint;
-                calcZoneSizing.DOASHighSetpoint = zoneSizingInput.DOASHighSetpoint;
-                calcZoneSizing.zoneSizingMethod = zoneSizingInput.zoneSizingMethod;
-                calcZoneSizing.zoneLatentSizing = zoneSizingInput.zoneLatentSizing;
-                calcZoneSizing.zoneRHDehumidifySetPoint = zoneSizingInput.zoneRHDehumidifySetPoint;
-                calcZoneSizing.zoneRHHumidifySetPoint = zoneSizingInput.zoneRHHumidifySetPoint;
-                calcZoneSizing.zoneRHDehumidifySchIndex = zoneSizingInput.zoneRHDehumidifySchIndex;
-                calcZoneSizing.zoneRHHumidifySchIndex = zoneSizingInput.zoneRHHumidifySchIndex;
-                calcZoneSizing.ZnLatCoolDgnSAMethod = zoneSizingInput.ZnLatCoolDgnSAMethod;
-                calcZoneSizing.LatentCoolDesHumRat = zoneSizingInput.LatentCoolDesHumRat;
-                calcZoneSizing.CoolDesHumRatDiff = zoneSizingInput.CoolDesHumRatDiff;
-                calcZoneSizing.ZnLatHeatDgnSAMethod = zoneSizingInput.ZnLatHeatDgnSAMethod;
-                calcZoneSizing.LatentHeatDesHumRat = zoneSizingInput.LatentHeatDesHumRat;
-                calcZoneSizing.HeatDesHumRatDiff = zoneSizingInput.HeatDesHumRatDiff;
-            }
+
+            if (DesDayNum == 1 && ZoneSizNum == 0) { // LKL I think this is sufficient for warning -- no need for array
+                if (!state.dataGlobal->isPulseZoneSizing) {
+                    ShowWarningError(
+                        state,
+                        format("SetUpZoneSizingArrays: Sizing for Zone=\"{}\" will use Sizing:Zone specifications listed for Zone=\"{}\".",
+                               state.dataZoneEquip->ZoneEquipConfig(CtrlZoneNum).ZoneName,
+                               zoneSizingInput.ZoneName));
+                }
+            }
+
+            zoneSizing.ZnCoolDgnSAMethod = zoneSizingInput.ZnCoolDgnSAMethod;
+            zoneSizing.ZnHeatDgnSAMethod = zoneSizingInput.ZnHeatDgnSAMethod;
```

Comment:
A few more nits. When the struct itself is something that corresponds to a zone, you don't need `zone` or `Zn` in the field name.